roles/grafana/tasks/main.yml

```diff
@@ -15,6 +15,14 @@
     name: "{{ __grafana_packages + __grafana_packages_extra }}"
     state: present
 
+- name: Template Grafana configuration
+  template:
+    src: grafana.ini.j2
+    dest: /etc/grafana/grafana.ini
```

Comment:
Should this be a (new) "{{ __grafana_conf }}" variable defined in vars/main.yml ...?

Reply:
done